src/components/ShareProfile/ShareUserProfile.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import qrCode from 'qr-code-and-vcard/dist/QrCode'
+import styled from 'styled-components/macro'
+
+export default function ShareUserProfile({ userDataInfo }) {
```

Comment:
Why not simply userData, or user as a name? Think about spreading it like so
```
<ShareUserProfile {...user}/>
```
and destructure it in the function parameter as ususal.

This IS a QR code, maybe think of a name that communicates this.